support_branding/views/res_config_settings.xml

```diff
@@ -0,0 +1,96 @@
+                                        <br />
+
+                                        <!-- URL -->
+                                        <label for="support_company_url" />
+                                        <div class="text-muted">
+                                            Add the support company URL
+                                            e.g https://odoo-community.org
+                                        </div>
+                                        <div class="content-group">
+                                            <div class="mt16">
+                                                <field name="support_company_url" />
+                                            </div>
+                                        </div>
+                                        <br />
+                                        <!-- Branding Color -->
+                                        <label for="support_branding_color" />
+                                        <div class="text-muted">
+                                            Add a branding color
+                                            e.g #b163a3
+                                        </div>
+                                        <div class="content-group">
+                                            <div class="mt16">
+                                                <field name="support_branding_color" />
+                                            </div>
+                                        </div>
+                                        <br />
+                                        <!-- Email -->
+                                        <label for="support_email" />
+                                        <div class="text-muted">
+                                            Add the support email to send issues
+                                            e.g info@helpdesksupport.com
+                                        </div>
+                                        <div class="content-group">
+                                            <div class="mt16">
+                                                <field name="support_email" />
+                                            </div>
+                                        </div>
+                                        <br />
+                                         <!-- Release -->
+                                        <label for="support_release" />
+                                        <div class="text-muted">
+                                            Add the support release
+                                        </div>
+                                        <div class="content-group">
+                                            <div class="mt16">
+                                                <field name="support_release" />
+                                            </div>
+                                        </div>
+                                        <br />
```

Comment:
Was thinking to make it easy to test this module we can add a button here that says `Test exception` which then calls something that generates an exception eg just a function with `raise UserError('Test exception')`

Reply:
added a js test for this and also a button function to generate division by zero error so that users can test. See below results of the test.

Reply:
Nice! :-)